library/utils.css

```diff
@@ -57,6 +57,13 @@
 p {
   line-height: 2rem;
 }
+
+.cta-text {
+  background-color: var(--primary-color);
+  color: var(--text-dark-color);
+  font-weight: 800;
+  padding: 0.2rem;
```

Comment:
can be used px

Reply:
Will be refactored when doing documentation site.